src/nvim/testdir/shared.vim

```diff
@@ -262,6 +266,12 @@ func! Screenline(lnum)
   return matchstr(line, '^.\{-}\ze\s*$')
 endfunc
 
-func CanRunGui()
-  return has('gui') && ($DISPLAY != "" || has('gui_running'))
+" Gets the text of a terminal line, using term_scrape()
+func Get_terminal_text(bufnr, row)
+  let list = term_scrape(a:bufnr, a:row)
+  let text = ''
+  for item in list
+    let text .= item.chars
+  endfor
+  return text
 endfunc
```

Comment:
We should not add `Get_terminal_text` (unused, and removed in Vim via "patch 8.1.0769: :stop is covered in two tests" later).  Merged already via https://github.com/neovim/neovim/pull/10157.

Reply:
It's harmless and patch 8.1.1581 will remove it.